linera-views/src/common.rs

```diff
@@ -122,6 +125,121 @@ where
     }
 }
 
+/// GreatestLowerBoundIterator iterates over the entries of a BTreeSet.
+/// The function call `get_lower_bound(val)` returns a `Some(x)` where `x` is the highest
+/// entry such that `x <= val`. If none exists then None is returned.
```

Comment:
The public interface is now `is_index_absent` instead of `get_lower_bound`, so that's probably what should be documented.
Or you could remove these lines and put the documentation on the method itself.

Reply:
Ok, I documented both functions.